ApprovalTests.Swift/Core/Options.swift

```diff
@@ -63,7 +63,7 @@ public struct Options {
         data[.scrubber] as? Scrubber ?? ScrubNothing()
     }
 
-    public func with(namerCreator: @escaping (String) -> ApprovalNamer) -> Options {
+    public func with(namerCreator: @escaping (String, String) -> ApprovalNamer) -> Options {
```

Comment:
What are the arguments of the closure here? You could name them to make this more obvious. Perhaps even create a type alias for the closure.

Reply:
This reveals a deeper need (beyond this change) for a missing type.